src/testcase_simple_phys.F

```diff
@@ -453,8 +456,9 @@ subroutine simplerad(rho0,zh,zf,dum1,dum2,qt  ,ql  ,qa,frad,zir)
         frad(i,j,k) = fr1+fr2+fr3
       enddo
       enddo
-      enddo
       !$acc end parallel
+      enddo
+
       stop 'simplerad: at the end of the subroutine'
```

Comment:
Probably can remove the stop statement from this subroutine.

Reply:
Sure and I think this subroutine is not called in our current test.